nav2_util/include/nav2_util/simple_action_client.hpp

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2018 Intel Corporation
```

Comment:
I've removed the simple_action_client template since this turned out to not be very useful over using the action client interface directly. So, this file has been removed.